### DISCUSSION TITLE
[A-L] (10 Febbraio 2023) Esercizio 6

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#377] [Esami A-L]
Dimostrare per Induzione che, per ogni $\Large n \geq 1$,

$$
    \Large \sum_{i =1}^{n} \frac{1}{i(i + 1)} = \frac{n}{n + 1}.
$$

Specificare il Caso Base, l’Ipotesi Induttiva e la dimostrazione del Passo Induttivo.

**Comment by DavideGalilei** [comment]
Soluzione già pubblicata su https://github.com/sapienzastudentsnetwork/metodi-matematici-per-l-informatica/discussions/408#discussioncomment-12279580
